doc/tutorials/objdetect/traincascade.markdown

```diff
@@ -184,146 +103,115 @@ img/img2.jpg  2  100 200 50 50   50 30 25 25
 
-Command line arguments of opencv_traincascade application grouped by purposes:
+Using the tool is quite straightforward. The tool accepts several required and some optional parameters:
 
--#  Common arguments:
+ - `--annotations` <b>(required)</b> : path to annotations txt file, where you want to store your annotations, which is then passed to the `-info` parameter [example - /data/annotations.txt]
+ - `--images` <b>(required)</b> : path to folder containing the images with your objects [example - /data/testimages/]
+ - `--maxWindowHeight` <i>(optional)</i> : if the input image is larger in height then the given resolution here, resize the image for easier annotation, using `--resizeFactor`.
+ - `--resizeFactor` <i>(optional)</i> : factor used to resize the input image when using the `--maxWindowHeight` parameter.
 
-    -   -data \<cascade_dir_name\>
+Note that the optional parameters can only be used together. An example of a command that could be used can be seen below
 
-        Where the trained classifier should be stored.
-
-    -   -vec \<vec_file_name\>
-
-        vec-file with positive samples (created by opencv_createsamples utility).
-
-    -   -bg \<background_file_name\>
-
-        Background description file.
-
-    -   -numPos \<number_of_positive_samples\>
-    -   -numNeg \<number_of_negative_samples\>
-
-        Number of positive/negative samples used in training for every classifier stage.
-
-    -   -numStages \<number_of_stages\>
-
-        Number of cascade stages to be trained.
-
-    -   -precalcValBufSize \<precalculated_vals_buffer_size_in_Mb\>
-
-        Size of buffer for precalculated feature values (in Mb).
-
-    -   -precalcIdxBufSize \<precalculated_idxs_buffer_size_in_Mb\>
-
-        Size of buffer for precalculated feature indices (in Mb). The more memory you have the
-        faster the training process.
-
-    -   -baseFormatSave
-
-        This argument is actual in case of Haar-like features. If it is specified, the cascade will
-        be saved in the old format.
-
-    -   -numThreads \<max_number_of_threads\>
-
-        Maximum number of threads to use during training. Notice that the actual number of used
-        threads may be lower, depending on your machine and compilation options.
-
-    -   -acceptanceRatioBreakValue \<break_value\>
-
-        This argument is used to determine how precise your model should keep learning and when to stop.
-        A good guideline is to train not further than 10e-5, to ensure the model does not overtrain on your training data.
-        By default this value is set to -1 to disable this feature.
-
--#  Cascade parameters:
-
-    -   -stageType \<BOOST(default)\>
+@code{.text}
+opencv\_annotation --annotations=/path/to/annotations/file.txt --images=/path/to/image/folder/`
```

Comment:
Let's replace all `opencv\_` with `opencv_`. I think it will work fine.
On this line there is backtick at the end.